pinc/Pool.inc

```diff
@@ -58,16 +54,9 @@ class Pool extends Stage
             "tools/pool.php?pool_id=$id"
         );
 
-        $this->project_checkedout_state = $project_checkedout_state;
-        $this->project_available_state = $project_available_state;
         $this->foo_Header = $foo_Header;
         $this->foo_field_name = $foo_field_name;
         $this->blather = $blather;
-
-        $this->states = [
```

Comment:
Ditto the review made in `Round.inc` about a breadcrumb comment.

Reply:
Done this also.